test/correctness/mongo_model.py

```diff
@@ -447,13 +447,17 @@ def __init__(self, options, collectionname=''):
             '$push': self.process_update_operator_push,
             '$bit': self.process_update_operator_bit,
         }
+        self.indexes = []
 
     def remove(self):
         self.data = dict()
 
     def _insert(self, doc):
         if '_id' not in doc:
             doc['_id'] = gen.random_object_id()
+        tmp = self.data.values() + [doc]
```

Comment:
We should add a similar code block for updates.

Reply:
Done